rust/src/lib.rs

```diff
@@ -34,54 +48,66 @@ impl GraphWrapper {
         }
     }
 
-    pub fn get_modules(&self) -> FxHashSet<String> {
+    pub fn get_modules(&self) -> HashSet<String> {
         self._graph
-            .get_modules()
-            .iter()
-            .map(|module| module.name.clone())
+            .all_modules()
```

Comment:
Any reason why we're renaming this method? Maybe easier to have the same name as on the wrapper?

Reply:
Happy to change this - I think it's a copy over from pyimports.